adapters/adview/adview.go

```diff
@@ -58,6 +59,22 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		}
 	}
 
+	// Check if imp comes with bid floor amount defined in a foreign currency
+	if imp.BidFloor > 0 && imp.BidFloorCur != "" && strings.ToUpper(imp.BidFloorCur) != "USD" {
+		// Convert to US dollars
+		convertedValue, err := requestInfo.ConvertCurrency(imp.BidFloor, imp.BidFloorCur, "USD")
```

Comment:
Pull request #2071 makes possible for adapters to include their own currency conversion rates in their JSON tests. When said PR gets merged, a test case could be included to add test coverage to lies 64 to 72

Reply:
i add the case currency_rate_not_found.json, but it seems cause validate error, so i remove it as temporary. so did i forgot something?